index.html

```diff
@@ -4027,6 +4105,84 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="header">
+			<rdef>header</rdef>
+			<div class="role-description">
+				<p>A set of user interface objects and information that represents a collection of introductory items for the element's closest ancestral content group. For instance, a <code>header</code> can include the heading, introductory statement and related meta data for a section of content, for instance a <rref>region</rref> or <rref>article</rref>, within a web page.</p>
+				<p>A <code>header</code> does not represent site-oriented or globally repeating content found across multiple pages of a website. For such content, the <rref>banner</rref> role would be more appropriate.</p>
```

Comment:
Maybe "sectionheader" to be more explicit that this is about sub-sections rather than page headers?

Ditto for footer.